src/commands/read.ts

```diff
@@ -44,8 +44,7 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
   const options: ConnectOptions = {
     chain,
   };
-  const tbl = await connect(options);
-  const res = await tbl.read(query);
+  const res = await connect(options).read(query);
```

Comment:
not async anymore!

Reply:
well, the connect part isn't async anymore, the read call obviously is

Reply:
Nice!